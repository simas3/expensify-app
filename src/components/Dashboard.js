import React from 'react'
import ExpenseList from './ExpenseList'
import ExpenseListFilters from './ExpenseListFilters'


const Dashboard = () => (
  <div className="container">
    <ExpenseListFilters />
    <ExpenseList />
  </div>
)


export default Dashboard;