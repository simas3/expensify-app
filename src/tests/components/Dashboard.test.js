import React from 'react'
import ReactShallowRenderer from 'react-test-renderer/shallow'
import Dashboard from '../../components/Dashboard'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json';

test('should render Dashboard component', () => {
    const wrapper = shallow(<Dashboard />)
    expect(toJson(wrapper)).toMatchSnapshot()
})