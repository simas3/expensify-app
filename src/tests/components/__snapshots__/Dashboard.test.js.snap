// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Dashboard component 1`] = `
<div
  className="container"
>
  <Connect(ExpenseListFilers) />
  <Connect(Component) />
</div>
`;
